magento/pwa-studio
# PWA-1951: Pagebuilder image content-type: mobile image is visible on desktop

- Updated logic to correctly pass desktop and mobile image src data in the config aggregator
- Added css class to hide mobile only image
- Updated test coverage



## Description

PWA pagebuilder image component should support mobile image only setting

*Steps to Reproduce*
 # Create cms page and add image content-type
 # Set only mobile image for the for content-type
 # Visit cms page on Venia

*Expected*
 (/) Image is not visible on the desktop, but visible on mobile screen width only
*Actual*
 (x) Image is visible on both desktop and mobile screen widths

 *Test Plan*
 * Verify mobile image is not visible in desktop on Venia CMS page
 * Verify mobile image is visible on mobile on Venia CMS page
 * Verify this use case is covered with an automated test

## Related Issue







Closes #[PWA-1951](https://jira.corp.magento.com/browse/PWA-1951)
## Acceptance





### Verification Stakeholders



### Specification




## Verification Steps







#### Test scenario(s) for direct fix/feature




 *Test Plan*
 * Verify mobile image is not visible in desktop on Venia CMS page
 * Verify mobile image is visible on mobile on Venia CMS page
 * Verify this use case is covered with an automated test

#### Test scenario(s) for any existing impacted features/areas





#### Test scenario(s) for any Magento Backend Supported Configurations





#### Is Browser/Device testing needed?





#### Any ad-hoc/edge case scenarios that need to be considered?




## Screenshots / Screen Captures (if appropriate)

## Breaking Changes (if any)





## Checklist



-   I have added tests to cover my changes, if necessary.
-   I have added translations for new strings, if necessary.
-   I have updated the documentation accordingly, if necessary.
